libs/profile/src/lib/components/profile/profile.component.ts

```diff
@@ -56,14 +65,20 @@ export class ProfileComponent {
                     ) {
                         this.authService.changePassword(result.data.newPassword, result.data.oldPassword);
```

Comment:
Hier müssen wir noch im authService in der changePassword() handeln dass evt. das falsche Passwort eingegeben wurde

Reply:
ist umgesetzt